docs/site/express-with-lb4-rest-tutorial.md

```diff
@@ -0,0 +1,358 @@
+? Property type: number
+? Is id the ID property? Yes
+? Is it required?: No
+? Default value [leave blank for none]:
+
+Let's add another property to Note
+Enter an empty property name when done
+
+? Enter the property name: title
+? Property type: string
+? Is it required?: Yes
+? Default value [leave blank for none]:
+
+Let's add another property to Note
+Enter an empty property name when done
+
+? Enter the property name: content
+? Property type: string
+? Is it required?: No
+? Default value [leave blank for none]:
+
+Let's add another property to Note
+Enter an empty property name when done
+
+? Enter the property name:
+
+   create src/models/note.model.ts
+   update src/models/index.ts
+
+Model note was created in src/models/
+```
+
+#### Add a DataSource
+
+Now, let's create a simple in-memory datasource by running the `lb4 datasource`
+command and filling out the prompts as follows:
+
+```sh
+$ lb4 datasource
+? Datasource name: ds
+? Select the connector for ds: In-memory db (supported by StrongLoop)
+? window.localStorage key to use for persistence (browser only):
+? Full path to file for persistence (server only): ./data/ds.json
+
+  create src/datasources/ds.datasource.json
+  create src/datasources/ds.datasource.ts
+  update src/datasources/index.ts
+
+Datasource ds was created in src/datasources/
+```
+
+Similar to the `Todo` example, let's create the `ds.json` by creating a data
+folder at the application's root.
+
+```sh
+$ mkdir data
+$ touch data/ds.json
+```
+
+Then copy and paste the following into the `ds.json` file:
+
+```json
+{
+  "ids": {
+    "Note": 3
+  },
+  "models": {
+    "Note": {
+      "1": "{\"title\":\"Things I need to buy\",\"content\":\"milk, cereal, and waffles\",\"id\":1}",
+      "2": "{\"title\":\"Great frameworks\",\"content\":\"LoopBack is a great framework\",\"id\":2}"
+    }
+  }
+}
+```
+
+#### Add Note Repository
+
+To create the repository, run the `lb4 repository` command and choose the
+`DsDataSource` and `Note` model, as follows:
+
+```sh
+$ lb4 repository
+? Please select the datasource DsDatasource
+? Select the model(s) you want to generate a repository Note
+   create src/repositories/note.repository.ts
+   update src/repositories/index.ts
+
+Repository Note was created in src/repositories/
+```
+
+#### Add Note Controller
+
+To complete the `Note` application, create a controller using the
+`lb4 controller` command and the following prompts:
+
+```sh
+$ lb4 controller
+? Controller class name: note
+? What kind of controller would you like to generate? REST Controller with CRUD functions
+? What is the name of the model to use with this CRUD repository? Note
+? What is the name of your CRUD repository? NoteRepository
+? What is the type of your ID? number
+? What is the base HTTP path name of the CRUD operations? /notes
+   create src/controllers/note.controller.ts
+   update src/controllers/index.ts
+
+Controller note was created in src/controllers/
+```
+
+### Express Class
```

Comment:
The title should be more meaningful, such as `Create a facade Express application`